.github/workflows/server.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         with:
           java-version: "12.0.2"
           architecture: x86
-      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.2.0
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
1. Should pin to a specific version for reproducibility.
2. `GoogleCloudPlatform/...` has been deprecated:
https://github.com/google-github-actions/setup-gcloud/tree/99f25f16fd0e9911d1ea055b84b3d43e793ae6a5/setup-gcloud#-deprecation-notice